inventory/group_vars/all.yml

```diff
@@ -10,6 +10,10 @@ local_release_dir: "/tmp/releases"
 # Random shifts for retrying failed ops like pushing/downloading
 retry_stagger: 5
 
+# Controll delayed startup/restart for services, in seconds
```

Comment:
I'd prefer to keep them here. Otherwise there are 5 roles to dupicate those in defaults of each!